README.md

```diff
@@ -38,48 +38,58 @@ If you would like to get involved, please also consider joining our
 
 -----
 
-## How to update the parser and dictionary
+## How tos
 
-To build the dictionary and parser locally, you must have NodeJS and NPM
-[installed](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
-Open a terminal in the `node` folder and run once the following command:
+### How to update a book
 
-```
-npm i
-```
+Ensure you have [Rust] installed on your machine.
 
-After every change in the `node` folder or in the dictionary, run the following command to build:
+[Rust]: https://www.rust-lang.org/tools/install
 
-```
-npm run build
-```
+1. Install the dependencies
+  ```
+  cargo install mdbook
+  cargo install mdbook-linkcheck
+  cargo install mdbook-regex-replacer
+  ```
+2. Open a terminal and navigate one of the books folders (e.g., `books/refgram`).
+3. Run `mdbook serve`.
 
-To open the web pages in your browser you need to run an HTTP server in the
-root folder, for exemple using [`miniserve`]. You can then
-browse the local web site at http://127.0.0.1:8080/index.html .
+The dev server is now running. Copy the localhost address and paste it into your
+browser to see your local copy of the website.
 
-To add a new word in the dictionary, you must edit `dictionary/en.yaml` and
-create a new entry for your word. You should set the `id` field to
-`INSERT_WORD_ID`, and then run the command `npm run ids` in the `node` folder so
-that `INSERT_WORD_ID` is automatically replaced with a random unique ID.
+The dev server will update this local website after every change you make.
 
-[`miniserve`]: https://lib.rs/crates/miniserve
+[`mdbook`]: https://lib.rs/crates/mdbook
```

Comment:
Now unused. Would be great to put back intro that books are made with mdbook (which justified the link).